app/assets/stylesheets/layouts/vacancies/index.scss

```diff
@@ -1,4 +1,4 @@
-.vacancies_index {
+.vacancies_index, .schools_index {
```

Comment:
Although I get that you're trying to reuse the styles here, it seems a bit odd cos the file is in `app/assets/stylesheets/layouts/vacancies`.

Alex may know a smart SASSy way of reusing all of the styles here.